neurolib/models/hopf/timeIntegration.py

```diff
@@ -76,14 +77,14 @@ def timeIntegration(params):
     x_ou = params["x_ou"]
     y_ou = params["y_ou"]
 
-    x_ext = params["x_ext"]
-    y_ext = params["y_ext"]
-
     # state variable arrays, have length of t + startind
     # they store initial conditions AND simulated data
     xs = np.zeros((N, startind + len(t)))
     ys = np.zeros((N, startind + len(t)))
 
+    x_ext = ap.adjust_shape(params["x_ext"], xs)
+    y_ext = ap.adjust_shape(params["y_ext"], ys)
+
```

Comment:
This should not be necessary! Why is this duplicated? It should be done in `loadDefaultParams.py` only?